.dir-locals.el

```diff
@@ -0,0 +1,5 @@
+(
```

Comment:
Should be possible to derive these from either of the `hie.yaml` files.

Reply:
Ah, yeah, I'll delete the file.

Reply:
Nah, keep it in, it was more a hint of how to get there faster :-)

Reply:
The name of the file is not `dante`-specific unfortunately. It's `emacs`-specific, so people may include their own stuff in the same file (e.g. `intero` uses the same file, but puts different stuff in there), so the file has to be deleted.